modules/cluster-logging-vector-tech-preview.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-vector-tech-preview_{context}"]
+:FeatureName: Vector
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-vector"]
+= About Vector
+[role="_abstract"]
+Vector is an observability data pipeline currently offered as an alternative to the default collector for the {logging}.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+The following outputs are supported:
+
+--
+
+* `elasticsearch`. An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+
+* `kafka`. A Kafka broker. The `kafka` output can use an unsecured or TLS connection.
+
+* `loki`. Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
+
+
+[id="cluster-logging-enabling-vector"]
+= Enabling Vector
+[role="_abstract"]
+Vector is not enabled by default. Use the following steps to enable Vector on your {product-title} cluster.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+.Prerequisites
+
+* {product-title}: 4.10
+* {logging-title-uc}: 5.4
+* FIPS disabled
+
+.Procedure
+
+. Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:
++
+[source,terminal]
+----
+$ oc -n openshift-logging edit ClusterLogging instance
+----
+
+. Add annotation `logging.openshift.io/preview-vector-collector: enabled` to the `ClusterLogging` custom resource (CR).
+
+[source,yaml]
+----
+  apiVersion: "logging.openshift.io/v1"
+  kind: "ClusterLogging"
+  metadata:
+    name: "instance"
+    namespace: "openshift-logging"
+    annotations:
+      logging.openshift.io/preview-vector-collector: enabled
+  spec:
+    collection:
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+    elasticsearch:
+      nodeCount: 3
+      resources:
+        limits:
+          memory: 16Gi
+        requests:
+          cpu: "1"
+          memory: 16Gi
+      storage:
+        storageClassName: "gp2"
+        size: "200G"
+      redundancyPolicy: "SingleRedundancy"
+  visualization:
+    type: "kibana"
+    kibana:
+      resources:
+        limits:
+          memory: 736Mi
+        requests:
+          cpu: 100m
+          memory: 736Mi
+      replicas: 1
+  collection:
+    logs:
+      type: "vector"
+      fluentd: {}
+        resources:
+          limits:
+            memory: 736Mi
+          requests:
+            cpu: 100m
+            memory: 736Mi
+----
+
+[NOTE]
+====
+You do not have to specify a secret in the `ClusterLogForwarder` CR if Vector is forwarding to Loki using token authentication.
```

Comment:
Looking at [the code](https://github.com/openshift/cluster-logging-operator/blob/49b27f942cc7b6e23116de88d6c2daaf6c79557d/internal/generator/vector/output/loki/loki.go#L240-L255) it seems like token authentication is only possible using the token of the `logcollector` serviceaccount. I don't know if this is intentional yet.

Reply:
@xperimental User provided bearer token is not working currently, this is the bug https://issues.redhat.com/browse/LOG-2461 .So no need to create any secret with token and specify it in the ClusterLogForwarder instance. If an in-cluster Loki URL is detected in the ClusterLogForwarder, the token is fetched from the logcollector SA and added to the Vector config.

Reply:
Also the code for auto-fetch of the token when no secret provided is intentional.

Reply:
So admonition is staying in the vector section, removed from the loki section.